### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ASM995**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---report-a-bug.md [hunk 1]
```diff
@@ -26,13 +26,17 @@ If you can, add screenshots to show the problem. If you need to show a code exam
 ## 🖥 Desktop
  - OS: [e.g. macOS]
  - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
+ - Browser Version: [e.g. 22]
+- Website Version [eg. V1, V2]
```
Comment (GCHQ-Developer-847) [inline .github/ISSUE_TEMPLATE/---report-a-bug.md:30]:
```suggestion
 - Website Version: [e.g. V1, V2]
```
Same on line 38

Also not sure 'version' needs to be capitalised - but doesn't really matter either way

File: .github/ISSUE_TEMPLATE/---suggest-a-design-improvement-.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name: "✍️ Suggest a Design improvement "
 about: Suggest some design, research or investigation work.
 title: ''
-labels: ''
+labels: Design
```
Comment (GCHQ-Developer-847) [inline .github/ISSUE_TEMPLATE/---suggest-a-design-improvement-.md:5]:
```suggestion
labels: design
```
For consistency with the other labels?

**Review by GCHQ-Developer-847 — CHANGES_REQUESTED** [review]

**Comment by GCHQ-Developer-847** [comment]
I think the destination branch should be develop - otherwise main will have a change that's not in develop

File: .github/ISSUE_TEMPLATE/---report-a-bug.md (hunk 1 above)
Comment (GCHQ-Developer-465) [reply .github/ISSUE_TEMPLATE/---report-a-bug.md:30]: I think the capitalisation of V1, V2 is ok as on the site it is capitalised also, therefore makes sense to leave it as it is.

File: .github/ISSUE_TEMPLATE/---suggest-a-design-improvement-.md (hunk 2 above)
Comment (GCHQ-Developer-465) [reply .github/ISSUE_TEMPLATE/---suggest-a-design-improvement-.md:5]: I agree here also

File: .github/ISSUE_TEMPLATE/---report-a-bug.md (hunk 1 above)
Comment (GCHQ-Developer-847) [reply .github/ISSUE_TEMPLATE/---report-a-bug.md:30]: I was thinking just the whole word 'Version', but it's not really a huge deal - happy either way. I agree that V1, V2 should be capitalised though :)

Comment (GCHQ-Developer-112) [reply .github/ISSUE_TEMPLATE/---report-a-bug.md:30]: I agree that version shouldn't be capitalised if it's not the first word i.e. "Browser version", "Website version"

**Review by GCHQ-Developer-530 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/------suggest-a-development-implementation-improvement-.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name: "\U0001F9D1‍\U0001F4BB Suggest a Development/implementation improvement "
 about: Suggest something for the UI Kit or website code.
 title: ""
-labels: ""
+labels: development
```
Comment (GCHQ-Developer-112) [inline .github/ISSUE_TEMPLATE/------suggest-a-development-implementation-improvement-.md:5]:
```suggestion
labels: "development"
```

File: .github/ISSUE_TEMPLATE/---report-a-bug.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 name: "\U0001F41B Report a bug"
 about: Tell us about a problem with the UI Kit or Design System website.
 title: ""
-labels: ""
+labels: bug
```
Comment (GCHQ-Developer-112) [inline .github/ISSUE_TEMPLATE/---report-a-bug.md:5]:
```suggestion
labels: "type: bug 🐛"
```

File: .github/ISSUE_TEMPLATE/---suggest-a-design-improvement-.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 name: "✍️ Suggest a Design improvement "
 about: Suggest some design, research or investigation work.
 title: ""
-labels: ""
+labels: design
```
Comment (GCHQ-Developer-112) [inline .github/ISSUE_TEMPLATE/---suggest-a-design-improvement-.md:5]:
```suggestion
labels: "design"
```

**Review by GCHQ-Developer-112 — CHANGES_REQUESTED** [review]

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by GCHQ-Developer-465 — APPROVED** [review]

**Pull request merged**